concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-1084	1084	wohnen (zu Hause sein)	zu Hause sein	to be at home		
+Steinthal-1875-1565-1085	1085	Reise	Reise	journey	2393	JOURNEY
+Steinthal-1875-1565-1086	1086	Gepäck	Gepäck	baggage		
+Steinthal-1875-1565-1087	1087	Stab	Stab	cane	1296	WALKING STICK
+Steinthal-1875-1565-1088	1088	Wagen	Wagen	carriage	2620	CARRIAGE
+Steinthal-1875-1565-1089	1089	Peitsche	Peitsche	whip	2630	WHIP
+Steinthal-1875-1565-1090	1090	Zügel	Zügel	reins		
+Steinthal-1875-1565-1091	1091	Weg	Weg	way	2457	PATH OR ROAD
+Steinthal-1875-1565-1092	1092	Brücke	Brücke	bridge	1011	BRIDGE
+Steinthal-1875-1565-1093	1093	reiten	reiten	to ride	367	RIDE
+Steinthal-1875-1565-1094	1094	fahren	fahren	to drive	741	DRIVE
+Steinthal-1875-1565-1095	1095	übersetzen	übersetzen	to ferry over		
+Steinthal-1875-1565-1096	1096	absteigen	absteigen	to dismount		
+Steinthal-1875-1565-1097	1097	hinauf-, hinabsteigen	hinaufsteigen	to ascend	705	GO UP (ASCEND)
+Steinthal-1875-1565-1098	1098	hinauf-, hinabsteigen	hinabsteigen	to descend	2014	GO DOWN (DESCEND)
+Steinthal-1875-1565-1099	1099	ausruhen	ausruhen	to rest	168	REST
+Steinthal-1875-1565-1100	1100	zurückbleiben	zurückbleiben	to stay behind		
+Steinthal-1875-1565-1101	1101	entgegengehn	entgegengehn	to approach	1778	APPROACH
+Steinthal-1875-1565-1102	1102	begegnen	begegnen	to meet	1831	MEET
+Steinthal-1875-1565-1103	1103	sich trennen	sich trennen	to split up		
+Steinthal-1875-1565-1104	1104	grüssen	grüssen	to greet	3552	GREET
+Steinthal-1875-1565-1105	1105	Ceremonien beim Grüssen (etwa: Hand geben)	Ceremonien beim Grüssen	ceremonies of greeting		
+Steinthal-1875-1565-1106	1106	Ceremonien beim Grüssen (etwa: Hand geben)	Hand geben	to shake hands		
+Steinthal-1875-1565-1107	1107	Abschied nehmen	Abschied nehmen	to say farewell		
+Steinthal-1875-1565-1108	1108	sich auf der Reise irren	sich auf der Reise irren	to lose one‘s way		
+Steinthal-1875-1565-1109	1109	den Weg zeigen (überhaupt etwas zeigen)	den Weg zeigen	to show the way		
+Steinthal-1875-1565-1110	1110	den Weg zeigen (überhaupt etwas zeigen)	überhaupt etwas zeigen	to show something	1039	SHOW
+Steinthal-1875-1565-1111	1111	führen	führen	to lead	43	LEAD (GUIDE)
+Steinthal-1875-1565-1112	1112	schicken	schicken	to send	1127	SEND
+Steinthal-1875-1565-1113	1113	Bote	Bote	messenger		
+Steinthal-1875-1565-1114	1114	Er hat mich gut aufgenommen.	Er hat mich gut aufgenommen.	He accommodated me well.		
+Steinthal-1875-1565-1115	1115	an einem Orte verweilen	an einem Orte verweilen	to stay in one place		
+Steinthal-1875-1565-1116	1116	Du bist lange geblieben.	Du bist lange geblieben.	You have stayed long.		
+Steinthal-1875-1565-1117	1117	schneiden	schneiden	to cut	1432	CUT
+Steinthal-1875-1565-1118	1118	Ein Stock bricht.	Ein Stock bricht.	A stick breaks.		
+Steinthal-1875-1565-1119	1119	den Stock zerbrechen	den Stock zerbrechen	to break a stick		
+Steinthal-1875-1565-1120	1120	Er ist zerbrochen.	Er ist zerbrochen.	It is broken.	588	BROKEN
```

Comment:
"Er" ist zerbrochen?

Reply:
der Stock; I believe this was in relation to the previous two concepts

Reply:
Der Stock, wahrscheinlich.